ui/app/AppLayouts/Chat/ChatLayout.qml

```diff
@@ -27,12 +27,12 @@ StatusAppThreePanelLayout {
     property var store
 
     // Not Refactored
-//    property var messageStore
```

Comment:
@jrainville all commented parts in `ChatLayout` component are like that because of new (chat/community) model introduced on the backend and we still don't have appropriate support on the qml side for that. That will be a breaking change on the qml side, and that's why I commented out that part till we have that part appropriately done.

Reply:
I uncommented those for the purpose of testing the stickers. We can remove them again if you prefer, but then we can't test my changes